tests/sentry/api/endpoints/test_team_projects.py

```diff
@@ -385,7 +385,49 @@ def test_builtin_symbol_sources_unity(self):
         symbol_sources = ProjectOption.objects.get_value(
             project=unity_project, key="sentry:builtin_symbol_sources"
         )
-        assert symbol_sources == ["unity"]
+        assert symbol_sources == ["unity", "nvidia", "ubuntu"]
+
+    def test_builtin_symbol_sources_unreal(self):
+        """
+        Test that project option for builtin symbol sources contains relevant buckets
+        when creating a Unreal project, but uses defaults for other platforms.
+        """
+        response = self.get_success_response(
+            self.organization.slug,
+            self.team.slug,
+            name="unreal-app",
+            slug="unreal-app",
+            platform="unreal",
+            status_code=201,
+        )
+
+        unreal_project = Project.objects.get(id=response.data["id"])
+        assert unreal_project.platform == "unreal"
+        symbol_sources = ProjectOption.objects.get_value(
+            project=unreal_project, key="sentry:builtin_symbol_sources"
+        )
+        assert symbol_sources == ["nvidia", "ubuntu"]
```

Comment:
odd that only unreal fails as symbol_sources have the built-in ones (ios, microsoft, android, nuget) added to the result:

```
FAILED tests/sentry/api/endpoints/test_team_projects.py::TeamProjectsCreateTest::test_builtin_symbol_sources_unreal - AssertionError: assert ['ios', 'micr...oid', 'nuget'] == ['nvidia', 'ubuntu']
  
  At index 0 diff: 'ios' != 'nvidia'
  Left contains 2 more items, first extra item: 'android'
  
  Full diff:
    [
  -     'nvidia',
  -     'ubuntu',
  +     'ios',
  +     'microsoft',
  +     'android',
  +     'nuget',
    ]
```

These are included in all projects by default, not sure why only this one test breaks

Reply:
maybe:
[`98b141e` (#91796)](https://github.com/getsentry/sentry/pull/91796/commits/98b141e3b48510d788b6c5eb4bcefa18b270b4c6)